recipes/zlib/1.2.11/conanfile.py

```diff
@@ -48,7 +43,11 @@ def build(self):
 
     @property
     def _use_autotools(self):
+        if self.settings.os == "iOS":
```

Comment:
```suggestion
        if str(self.settings.os) in ["iOS", "watchOS", "tvOS"]:
```
as there is the same issue on other platforms

Reply:
good catch! I add this, and can, and will, test building it, hopefully already tomorrow in some time slot I can find 
but I can not run it on any device since I have no appple watch or apple tv, so it will not be as tested as the iOS part